### PR TITLE
chore: remove property_prefix macro

### PR DESCRIPTION
**Pull request by schalkneethling**
Removes `property_prefix` macro from remaining locales.

Part of #5603

### CODE REVIEW

**Review by mfuji09 — APPROVED** [review]
LGTM for `ja` file

File: files/ru/mozilla/firefox/releases/53/index.html [hunk 1]
```diff
@@ -157,9 +157,9 @@ <h3 id="HTML_XML"><font><font>HTML / XML</font></font></h3>
 <h3 id="CSS_2"><span><span><font><font>CSS</font></font></span></span></h3>
 
 <ul>
- <li><font><font>Удалены {{property_prefix ( "- Мос")}} приставкой вариант {{cssxref ( ": реж", ": каталог ()")}} псевдокласс ({{bug (1270406)}}).</font></font></li>
+ <li><font><font>Удалены <code>-moz</code> приставкой вариант {{cssxref ( ": реж", ": каталог ()")}} псевдокласс ({{bug (1270406)}}).</font></font></li>
  <li><code>-moz</code><font><font>Приставкой версия {{cssxref ( "текст-ALIGN-последний")}} был удалён ({{bug (1276808)}}).</font></font></li>
- <li><font><font>Удалены {{property_prefix ( "- Мос")}} приставкой вариант {{cssxref ( "известково ()")}} Метод ({{bug (1331296)}}).</font></font></li>
+ <li><font><font>Удалены <code>-moz</code> приставкой вариант {{cssxref ( "известково ()")}} Метод ({{bug (1331296)}}).</font></font></li>
```
Comment (sashasushko) [inline files/ru/mozilla/firefox/releases/53/index.html:162]:
```suggestion
 <li>Удалён вариант с префиксом <code>-moz</code> для метода {{cssxref("calc", "calc()")}} ({{bug(1331296)}}).</li>
```

File: files/ru/mozilla/firefox/releases/53/index.html [hunk 2]
```diff
@@ -157,9 +157,9 @@ <h3 id="HTML_XML"><font><font>HTML / XML</font></font></h3>
 <h3 id="CSS_2"><span><span><font><font>CSS</font></font></span></span></h3>
 
 <ul>
- <li><font><font>Удалены {{property_prefix ( "- Мос")}} приставкой вариант {{cssxref ( ": реж", ": каталог ()")}} псевдокласс ({{bug (1270406)}}).</font></font></li>
+ <li><font><font>Удалены <code>-moz</code> приставкой вариант {{cssxref ( ": реж", ": каталог ()")}} псевдокласс ({{bug (1270406)}}).</font></font></li>
  <li><code>-moz</code><font><font>Приставкой версия {{cssxref ( "текст-ALIGN-последний")}} был удалён ({{bug (1276808)}}).</font></font></li>
```
Comment (sashasushko) [inline files/ru/mozilla/firefox/releases/53/index.html:161]:
```suggestion
 <li>Удалён вариант с префиксом <code>-moz</code> для свойства {{cssxref("text-align-last")}} ({{bug(1276808)}}).</li>
```

**Review by sashasushko — COMMENTED** [review]
Fixed some translation

**Review by hochan222 — APPROVED** [review]
LGTM for `ko` 👍  🙇🙇

**Review by sashasushko — APPROVED** [review]
Thanks a lot 🤝 Approve from @mdn/yari-content-ru

**Comment by sashasushko** [comment]
@schalkneethling do you need help with resolving conflicts?

**Comment by schalkneethling** [comment]
> @schalkneethling do you need help with resolving conflicts?

Nope, I did not realize there were conflicts. Looking at it now, thanks.

**Comment by schalkneethling** [comment]
> @schalkneethling do you need help with resolving conflicts?

Conflicts addressed. Thanks for the nudge.

**Review by yin1999 — COMMENTED** [review]
approved by all those locales, going to merge this

**Pull request merged**